src/jobs/helm_push.yaml

```diff
@@ -36,4 +36,4 @@ steps:
 
         echo "Adding '<< parameters.charts_repo_url >>'"
         helm repo add --username "${CHARTS_REPO_USER}" --password "${CHARTS_REPO_PASS}" codacy << parameters.charts_repo_url >>
-        helm push ${HELM_DIR}/${CHART_NAME} codacy
+        helm push ${HELM_DIR}/${CHART_NAME} codacy --force
```

Comment:
This should have a parameter to disable it

Reply:
we only push to unstable and incubator, this should not affect the stable repo. Wouldn't it be enough? Even if it's a parameter it's kind of "static" since you would have to put it in the circleci configuration

Reply:
I agree with @DReigada. It would be nice to be able to disable this explicitly.

Reply:
We should only forcefully push to unstable, incubator should not allow force pushes.

Reply:
From that perspective I agree. I'll make a change, give me a few minutes